tests/foreman/api/test_role.py

```diff
@@ -214,12 +286,33 @@ def test_positive_create_overridable_filter(self):
             2. Override check is set to true
             3. Filter doesnt inherits taxonomies from role
 
-        :caseautomation: notautomated
-
         :CaseImportance: Critical
         """
+        role_name = gen_string('alpha')
+        role = entities.Role(
+            name=role_name,
+            organization=[self.role_org],
+            location=[self.role_loc]
+        ).create()
+        self.assertEqual(role.name, role_name)
+        dom_perm = entities.Permission(resource_type='Domain').search()
+        filtr = entities.Filter(
```

Comment:
any reason to use word `filtr` instead `filter`?

Reply:
FYI ... filter is reserved keyword in python

Reply:
ok